src/metabase/api/channel.clj

```diff
@@ -55,6 +55,9 @@
    details     :map
    active      [:maybe {:default true} :boolean]}
   (validation/check-has-application-permission :setting)
+  (when (t2/exists? :model/Channel :name name)
+    (throw (ex-info "Channel with that name already exists" {:status-code 400
```

Comment:
Since there is nothing malformed with the request, I would suggest using 409 (Conflict) instead. Worth checking with FE though.

Reply:
cc @npfitz

Reply:
I don't think that that would be a problem

Reply:
7e5c880c88 you got it